examples/huggingface-transformers/requirements.txt

```diff
@@ -1,4 +1,5 @@
 deepsparse
+onnxruntime>=1.4.0,<=1.6.0
```

Comment:
What's our current restriction on greater than 1.6?

Reply:
this is to match our `setup.py` this won't affect the example though so I'm okay taking it out